module/Application/src/Application/Controller/SessionRestController.php

```diff
@@ -54,7 +54,7 @@ public function create($data)
         if ($sessionOpenResponse['status'] == 201) {
             $this->securityLogger->loginSuccessful($sessionOpenResponse['body']['userId']);
         } elseif ($sessionOpenResponse['status'] == 401) {
-            $this->securityLogger->loginFailed($sessionOpenResponse['body']['error']);
+            $this->securityLogger->loginFailed($sessionOpenResponse['body']['error'], $sessionOpenResponse['body']['userId'] ?? null);
```

Comment:
Can we strip `userId` out of the response here to prevent information disclosure (or return it in a separate variable rather than the body)? I think you could use it for account enumeration if you authed directly against Membrane.